src/components/ImgEditor/ImgEditor.tsx

```diff
@@ -112,18 +112,22 @@ export default function ImgEditor<T extends FieldValues, K extends keyof T>(
       </div>
       <p className="text-xs text-gray-d1 dark:text-gray mt-2">
         <span>
-          Valid types are: JPG, JPEG, PNG and WEBP. Original uploaded image
-          should be less than 1MB in size.
-          <br />
-          {file
-            ? `Current (cropped) image size: ${humanize(
-                file.size / BYTES_IN_MB
-              )}MB.`
-            : ""}
+          Valid types are: JPG, JPEG, PNG and WEBP.{" "}
+          {maxSize ? (
+            <>
+              Image should be less than {maxSize / BYTES_IN_MB}MB in size.
+              <br />
+              {imgSize
```

Comment:
Current image size is only relevant if there is a maximum image size. 
@SovereignAndrey we can show this nevertheless of course, this is just a proposal.

Reply:
I can't imaging a world where we don't have _some form_  of upper limit on file size uploads. Even if large, it'd still need to exist.